src/Methods/Modular_Programming/Hard/Q1/MainTest.java

```diff
@@ -0,0 +1,56 @@
+package Methods.Modular_Programming.Hard.Q1;
```

Comment:
package names should all be lowercase

Reply:
Wait the package name is still incorrectly named? For clarity, I mean the packages, like the folders that this file is in